app/jobs/calendar_importer/event_resolver.rb

```diff
@@ -173,7 +174,7 @@ def room_number_strategy(place, address: nil)
       end
     end
 
-    return address, place
+    return place, address
```

Comment:
would it be better to return a hash to make it clearer the other end what's going on?

Reply:
Yeah, we could do that but it might make the already weird code that calls these functions look more obscure maybe. Currently it looks like 

```
      strategy = strategies[calendar.strategy]
      place, address = method(strategy).call(calendar.place)
```

which already feels kinda-obscure because Ruby doesn't really Like higher order functions